mongodbatlas/resource_mongodbatlas_project_ip_access_list.go

```diff
@@ -278,3 +283,54 @@ func resourceMongoDBAtlasIPAccessListImportState(ctx context.Context, d *schema.
 
 	return []*schema.ResourceData{d}, nil
 }
+
+func isProjectIPAccessInTheList(ctx context.Context, conn *matlas.Client, projectID, entry string) (bool, error) {
```

Comment:
[q] Could you show me where the bug was? I see that the main difference in your solution is that now we have `isProjectIPAccessInTheList` to check that the entry is in the project access list. Thanks!

Reply:
The bug was is that when you create 100 ips or more, it will fail with an error of 21 retries or something like that, I realized that happens because when you make a list, it returns maximum 100 items(if not specified in list options) but when you read TotalCount is more than 100, if the entry has not been found it's probably is in after 100 item, so it would never be found hence the error of 21 retries, so I made a function and use the List options so it if the first page the entry has not been found yet, will retry again with another page until is found(or if is last page)

Reply:
Thanks!